dstep/translator/Output.d

```diff
@@ -181,6 +215,112 @@ class Output
         singleLine(fmt, args);
     }
 
+    private Tuple!(string, string, string) adaptiveLineParts(Char, Args...)(
```

Comment:
I find a bit difficult to understand what all these "adaptive" functions are used for. Perhaps some documentation would help.